scripts/collector/postgres/collect-data.sh

```diff
@@ -274,12 +274,17 @@ then
 	DMA_SOURCE_ID="NA"
 fi
 
+PKEY="NA"
+
 ${SQLCMD}  --user=$user --password -h $host -w -p $port  --no-align <<EOF
 \set VTAG ${V_FILE_TAG}
 \set DMA_SOURCE_ID '\'${DMA_SOURCE_ID}\''
 \set DMA_MANUAL_ID '\'${V_MANUAL_ID}\''
 \i sql/op_collect.sql
 EOF
+specsOut="output/opdb__pg_db_machine_specs_${V_FILE_TAG}.csv"
+host=$(echo ${connectString} | cut -d '/' -f 4 | cut -d ':' -f 1)
+./db-machine-specs.sh $host ${PKEY} ${DMA_SOURCE_ID} ${V_MANUAL_ID} ${specsOut}
```

Comment:
```suggestion
./db-machine-specs.sh $host ${V_FILE_TAG} ${DMA_SOURCE_ID} ${V_MANUAL_ID} ${specsOut}
```
I think we can just do this based on some recent changes in the `mcupdates-postgres` branch.